tests/kb_test.py

```diff
@@ -1026,5 +1039,23 @@ def testJsonFiles_allFilesHaveCorrectCategories_testPasses(
             categories = json_data.get("categories", {})
             standard_categories = categories.get(category, [])
 
+            assert all(group_key in CATEGORY_GROUPS for group_key in categories.keys())
             for standard_category in standard_categories:
                 assert standard_category in expected_categories
+
+
+def testJsonFiles_allFilesHaveCategories_shouldBeValid() -> None:
```

Comment:
```suggestion
def testJsonFiles_whenAllFilesHaveCategories_theyShouldBeValid() -> None:
```

I would make it more readable.